.github/workflows/tests-polkadot-deps.yml

```diff
@@ -77,10 +86,10 @@ jobs:
 
     strategy:
       matrix:
-        image: ['latest-rc']
-        required: ['optional']
+        image: ['latest']
+        required: ['required']
         include:
-          - image: 'latest-develop'
+          - image: 'latest-rc'
```

Comment:
`latest-rc` is now `latest-master`. I though I merged the change back into `develop`...